src/js/disability-benefits/actions/index.js

```diff
@@ -163,94 +163,104 @@ export function submitFiles(claimId, trackedItem, files) {
   });
 
   return (dispatch) => {
-    const uploader = new FineUploaderBasic({
-      request: {
-        endpoint: `${environment.API_URL}/v0/evss_claims/${claimId}/documents`,
-        inputName: 'file',
-        customHeaders: {
-          'X-Key-Inflection': 'camel',
-          Authorization: `Token token=${sessionStorage.userToken}`
-        }
-      },
-      cors: {
-        expected: true,
-        sendCredentials: true
-      },
-      multiple: false,
-      callbacks: {
-        onAllComplete: () => {
-          if (!hasError) {
-            window.dataLayer.push({
-              event: 'claims-upload-success',
-            });
-            dispatch({
-              type: DONE_UPLOADING,
-            });
-            dispatch(setNotification({
-              title: 'We have your evidence',
-              body: `Thank you for filing ${trackedItem ? trackedItem.displayName : 'additional evidence'}. We'll let you know when we've reviewed it.`
-            }));
-          } else {
-            window.dataLayer.push({
-              event: 'claims-upload-failure',
-            });
-            dispatch({
-              type: SET_UPLOAD_ERROR
-            });
-            dispatch(setNotification({
-              title: 'Error uploading files',
-              body: 'There was an error uploading your files. Please try again',
-              type: 'error'
-            }));
-          }
-        },
-        onTotalProgress: (bytes) => {
-          bytesComplete = bytes;
-          dispatch({
-            type: SET_PROGRESS,
-            progress: calcProgress(totalFiles, totalSize, filesComplete, bytesComplete)
-          });
-        },
-        onComplete: () => {
-          filesComplete++;
-          dispatch({
-            type: SET_PROGRESS,
-            progress: calcProgress(totalFiles, totalSize, filesComplete, bytesComplete)
-          });
-        },
-        onError: (id, name, reason) => {
-          const errorCode = reason.substr(-3);
-          // this is a little hackish, but uploader expects a json response
-          if (!errorCode.startsWith('2')) {
-            hasError = true;
-          }
-          if (errorCode === '401') {
-            dispatch({
-              type: SET_UNAUTHORIZED
-            });
-          }
-        }
-      }
-    });
     dispatch(clearNotification());
     dispatch({
       type: SET_UPLOADING,
       uploading: true,
-      uploader
     });
     dispatch({
       type: SET_PROGRESS,
-      progress: filesComplete / files.length
+      progress: 0
```

Comment:
This is just for the initial state, correct? Why wasn't it set to 0 before? Looks good, but this is the one part I can't test (the actual file upload and by extension, the progress).